flow/core/kernel/scenario/traci.py

```diff
@@ -649,39 +649,41 @@ def generate_cfg(self, net_params, traffic_lights, routes):
         num_traffic_lights = len(list(traffic_lights.get_properties().keys()))
         if num_traffic_lights > 0:
             if traffic_lights.baseline:
-                tl_params = traffic_lights.actuated_default()
-                tl_type = str(tl_params['tl_type'])
-                program_id = str(tl_params['program_id'])
-                phases = tl_params['phases']
-                max_gap = str(tl_params['max_gap'])
-                detector_gap = str(tl_params['detector_gap'])
-                show_detector = tl_params['show_detectors']
-
-                detectors = {'key': 'detector-gap', 'value': detector_gap}
-                gap = {'key': 'max-gap', 'value': max_gap}
-
-                if show_detector:
-                    show_detector = {'key': 'show-detectors', 'value': 'true'}
-                else:
-                    show_detector = {'key': 'show-detectors', 'value': 'false'}
-
-                nodes = self._inner_nodes  # nodes where there's traffic lights
-                tll = []
-                for node in nodes:
-                    tll.append({
-                        'id': node['id'],
-                        'type': tl_type,
-                        'programID': program_id
-                    })
-
-                for elem in tll:
-                    e = E('tlLogic', **elem)
-                    e.append(E('param', **show_detector))
-                    e.append(E('param', **gap))
-                    e.append(E('param', **detectors))
-                    for phase in phases:
-                        e.append(E('phase', **phase))
-                    add.append(e)
+                if traffic_lights.all_type == 'actuated':
```

Comment:
Can we add a comment here? I don't know what it means for traffic_lights.all_type to be actuated. Does this mean all lights are actuated?

Reply:
Yes it does and comment is added